src/streams/StreamIcon.js

```diff
@@ -15,12 +15,11 @@ type Props = $ReadOnly<{|
 |}>;
 
```

Comment:
```js
type Props = $ReadOnly<{|
// ...
  isMuted: boolean
// ...
```
This component no longer needs `isMuted` in `Props`.

Reply:
Stream Icon component is used at other places also they are sending props if I remove from here typescript error will be there. Can you once look into it and what should I do.

Reply:
If a child component no longer needs a particular prop, we should no longer pass it down in the parent component. If the prop from the parent component changes, this will cause unintended re-rendering in the child component even if the child component doesn't express any visible changes.